src/bundle/Resources/config/routing.yml

```diff
@@ -459,6 +459,12 @@ ezplatform.translation.remove:
     defaults:
         _controller: 'EzPlatformAdminUiBundle:Translation:remove'
 
+ezplatform.content.update_main_translation:
```

Comment:
Note my above comment about naming - if you (and others) agree you have to fix route name and path as well.